README.md

```diff
@@ -5,14 +5,37 @@ Firefox Accounts DB Server
 
 ## Usage
 
+If you would like to run a FxA Auth DB Server, then you probably want to look at the
+[Fxa Auth DB MySql](http://github.com/mozilla/fxa-auth-db-mysql/) repo (for production usage) or the
+[Fxa Auth DB Mem](http://github.com/mozilla/fxa-auth-db-mem/) repo (for testing purposes). Each of these repos
+implement a storage backend and will install this repo when you run `npm install` since they each import the server
+from here.
+
+## Usage
+
 ```js
-var createServer = require('fxa-auth-db-server')
-var database = require('./db') // An object the implements the DB API described below
+// Require the dbServer. This includes dbServer.createServer() and the dbServer.errors object.
+var dbServer = require('fxa-auth-db-server')
+
+// require the database engine and pass it a logger and the dbServer.errors
```

Comment:
What is "and pass it a logger and the `dbServer.errors`" here?
I only see a lowly `require()` in the following line (but we do pass a `dbServer.errors` instance down below on L24.